lib/fluent/plugin/out_s3.rb

```diff
@@ -142,10 +146,12 @@ def configure(conf)
         @path_slicer = Proc.new {|path|
           Time.now.strftime(path)
         }
+        @hms_slicer = Time.now.strftime("%H%M%S")
```

Comment:
This value is never updated in `write`.
Is this expected?
All stored object same `@hms_slicer` value.